install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -11,3 +11,5 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
   (NULL, 'actionModuleUninstallAfter', 'Module uninstall after', 'This hook is called at the end of module uninstall process', '1'),
   (NULL, 'actionPresentProductListing', 'Product Listing Presenter', 'This hook is called before a product listing is presented', '1')
 ;
+
+ALTER TABLE `PREFIX_shop` ADD `color` VARCHAR(50) NULL DEFAULT NULL AFTER `name`;
```

Comment:
50 for a hexa color code ?

Reply:
Don't update the `db_structure.sql` ?

Reply:
It can also be a html color name, like `BlanchedAlmond` for example. I don't know what is and will be the longest color name in the future, a length of 50 seems safe.

About `db_structure.sql` I was going to write about it, but you're so fast ;)

I noticed that the `PREFIX_shop` table was added in the upgrade SQL file `1.5.0.0.sql`, but was not added to `db_structure.sql`, do you guys know if there is a specific reason or if it was not forgotten?

In my opinion it should be added to the structure file since the table is created at install, but since this table is linked to multistore I'm not really sure.

Reply:
You don't have put a colorpicker ?

Reply:
yes, the color picker allows to write manually a html color name instead of picking a color (it's even written in the tooltip description), for example on order state creation page:

`Les états seront mises en évidence dans cette couleur. Couleurs HTML uniquement. "lightblue", "#CC6600")`

Reply:
about `db_structure.sql` I understand, actually `ps_shop` is setup via doctrine at install, I'll update my PR, it's back in WIP.